lib/Backends/OpenCL/OpenCL.h

```diff
@@ -167,7 +168,11 @@ class OCLBackend final : public Backend {
                      cl_device_id device, llvm::ArrayRef<size_t> global,
                      llvm::ArrayRef<size_t> local,
                      std::vector<KernelLaunch> &kernelLaunches);
-
+  /// Set OpenCL \p kernel arguments using the buffer operands of the
+  /// instruction \p I. The first of these arguments should be passed to the \p
+  /// kernel at index \p nextArgIdx.
+  size_t setKernelArgsForBuffers(cl_kernel kernel, const glow::Instruction &I,
```

Comment:
Can you make this a static in the cpp file?  It doesn't look it needs to be exported.